### ISSUE TITLE
Add placeholder and aria-label for "open walkthrough"

### ISSUE THREAD

**Issue by neeleshb** [microsoft/vscode#190923]
Type: <b>Bug</b>

The "Open walk-through" dialog lacks a placeholder in the input box and an appropriate aria-label. 

VS Code version: Code 1.78.0 (252e5463d60e63238250799aef7375787f68b4ee, 2023-05-03T20:09:46.685Z)
OS version: Linux x64 6.3.11-1rodete1-amd64
Modes:
![placeholder lacking](https://github.com/microsoft/vscode/assets/2174646/389fc4b8-3b5f-4777-8249-0a02dd827ded)

Sandboxed: No

**Comment by VSCodeTriageBot** [comment]
Thanks for creating this issue! It looks like you may be using an old version of VS Code, the latest stable release is 1.81.1. Please try upgrading to the latest version and checking whether this issue remains.

Happy Coding!

**Comment by bhavyaus** [comment]
This is fixed in newer builds:
![Image](https://github.com/microsoft/vscode/assets/25044782/718a4f89-cd3a-48f9-9005-49e178e462a2)
